packages/transformer/src/IModelTransformer.ts

```diff
@@ -2422,6 +2422,8 @@ export class IModelTransformer extends IModelExportHandler {
       } as SourceAndTarget);
 
     // FIXME: make importer.deleteRelationship not need full props
+    // FIXME<NICK>: Do we care that its possible an importer may want the full relationshipProps or should we just have them query for the full relationshipProps if they wanted them?
```

Comment:
if they want it, they should query it themselves, we should avoid unnecessary information at the API level.

Unfortunately in reality, the JavaScript query API is so slow that it may less performant today. But hopefully that is addressed and eventually this change will enable a performance optimization.

Reply:
I changed it, let me know your thoughts.

Could you explain more what you mean by "the JavaScript query API is so slow that it may less performant today". Less performant today as a result of passing all of the props? Which we're correcting in this PR now?

Reply:
I mean that `db.query("SELECT Prop1, Prop2, Prop3 FROM bis.ElementRefersToElements WHERE ECInstanceId=?")` may be slower `than db.relationships.getInstance(id)` in all cases due to `db.query`, `db.withPreparedStatement` and `db.createQueryReader` all being slower than native today even when doing less work.

Reply:
also if they are aware of that, they can just use `getInstance` themselves